README.md

```diff
@@ -244,3 +244,52 @@ The output looks similar to:
 ```text
 206.189.231.90
 ```
+
+### Failing over gateways
+
+In order to protect against gateway failures, it is recommended to prepare a standby gateway droplet and failover when necessary. While the operator does not support true HA, failing over enables to minimize the window of disruption.
+
+Requirements:
+
+1. Have one (or more) gateway droplets fully configured and ready to take traffic. (This may entail allow-listing the droplet's IP address with downstream receivers.)
+1. Implement a method to identify when a gateway droplet is failing. This could be as simple as probing the droplet repeatedly from inside the cluster and considering it down if the probes fail repeatedly, to aggregating multiple signals from observability systems and coming to an availability determination. The exact implementation depends on the user's needs and should ideally yield a good signal-to-noise ratio.
+1. All operator instances are up and running correctly at the time of the failover. (More specifically, at step (3) of the failover procedure outlined below.)
+
+Let's assume that we have a single destination IP address 34.160.111.145 that the active gateway with IP addess 10.116.0.5 is transmitting traffic for. The corresponding `StaticRoute` would look like this:
+
+```yaml
+apiVersion: networking.digitalocean.com/v1
+kind: StaticRoute
+metadata:
+  name: static-route-ifconfig.me
```

Comment:
I'd also reccommend changing the name to make it more clear that these are meant to be two static routes, rather than changing/editing the static route.
Something like `primary`
 It could result in users running `kubectl apply` instead of `kubectl delete`...`kubectl create`